GoogleDataTransport/GDTCORLibrary/GDTCOREvent.m

```diff
@@ -56,8 +56,10 @@ - (NSUInteger)hash {
   // This loses some precision, but it's probably fine.
   NSUInteger mappingIDHash = [_mappingID hash];
   NSUInteger timeHash = [_clockSnapshot hash];
-  NSUInteger dataObjectTransportBytesHash = [_dataObjectTransportBytes hash];
-  return mappingIDHash ^ _target ^ dataObjectTransportBytesHash ^ _qosTier ^ timeHash;
+  NSInteger dataObjectHash = [_dataObject hash];
```

Comment:
if it's a property, use dot access:  _dataObject.hash

Reply:
Doudou is just being consistent with what I do in other places. Since it's intra-class, ivar access is faster and the consequence is having KVO not work (which is fine, IMO).